src/llt/src/lib.rs

```diff
@@ -163,10 +167,10 @@ impl Drop for Entry {
 }
 
 #[cfg(unix)]
-pub type Thread = libc::pthread_t;
+pub type ThreadHandle = libc::pthread_t;
```

Comment:
I don't think `*Handle` is a valid word in the `pthread` world. So better to use something like `RawThread` or `OsThread`.